sdk/src/trace/tracer.cc

```diff
@@ -47,8 +47,13 @@ nostd::shared_ptr<trace_api::Span> Tracer::StartSpan(
   }
   else
   {
-    auto span = nostd::shared_ptr<trace_api::Span>{new (std::nothrow) Span{
-        this->shared_from_this(), processor_.load(), name, attributes, options}};
+    // TODO: Get parent span context from current context. For now we assume all spans
+    // have no parent, and pass an invalid parent span context to the Span constructor.
+    const trace_api::SpanContext kInvalidParentSpanContext(false, false);
```

Comment:
Can you please raise a ticket to fix this temporary change once the other PR is merged ?

Reply:
There's an issue to track the overall work for setting ids here: https://github.com/open-telemetry/opentelemetry-cpp/issues/285. I added a comment to give an update on the remaining work that needs to be done. Or, would it be better to create a new issue?